### PR TITLE
Release nf-core/pixelator 1.2.0

### PR DESCRIPTION
**Pull request by fbdtemme**
Dev -> Master for 1.2.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,30 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[1.2.0](https://github.com/nf-core/pixelator/releases/tag/1.2.0)] - 2024-05-??
```
Comment (johandahlberg) [inline CHANGELOG.md:6]: Just writing something here so that we remember to update the release date.

File: conf/test.config [hunk 2]
```diff
@@ -23,7 +23,8 @@ params {
     max_memory = '6.GB'
     max_time   = '6.h'
 
-    input  = "https://raw.githubusercontent.com/nf-core/test-datasets/pixelator/samplesheet/samplesheet.csv"
+    // Input data
+    input          = params.pipelines_testdata_base_path + 'pixelator/samplesheet/samplesheet.csv'
     input_basedir  = "https://raw.githubusercontent.com/nf-core/test-datasets/pixelator/testdata"
```
Comment (maxulysse) [inline conf/test.config:28]:
```suggestion
    input_basedir  = params.pipelines_testdata_base_path + 'pixelator/testdata'
```

You should be able to do something like that (untested)

**Review by maxulysse — APPROVED** [review]
Looks pretty straight forward to me

**Review by ramprasadn — APPROVED** [review]
LGTM 🚀

**Comment by fbdtemme** [comment]
Thanks everyone for the reviews!

**Pull request merged**